math/Cargo.toml

```diff
@@ -17,6 +17,7 @@ serde_json = { version = "1.0", default-features = false, features = [
 proptest = { version = "1.1.0", optional = true }
 winter-math = { package = "winter-math", version = "0.6.4", default-features = false, optional = true }
 miden-core = { package = "miden-core", version = "0.7", default-features = false, optional = true }
+rand = { version = "0.8.5", features = ["std"] }
```

Comment:
it could be better to use the same version as in the root Cargo.toml file:
```
rand = { workspace = true, features = ["std"] }
```

Reply:
Do you require the `std` feature or is it just copied from the `dev-dependencies` section? `no_std` builds break because of this feature.